docs/installation.rst

```diff
@@ -6,7 +6,7 @@ Installation
 
 ----
 
-Python and associated packages for science are freely available. Convenient science python package setups are available from `Enthought <https://store.enthought.com>`_ and `Continuum Analytics <http://continuum.io/downloads>`_. Enthought also includes an IDE, though there are a number of choices. Core science packages such as numpy, scipy, matplotlib, pandas and many others may also be installed directly via pip or your favorite package manager. 
+Python and associated packages for science are freely available. Convenient science python package setups are available from `Enthought <https://store.enthought.com>`_ and `Continuum Analytics <http://continuum.io/downloads>`_. Enthought also includes an IDE, though there are a number of choices. Core science packages such as numpy, scipy, matplotlib, pandas and many others may also be installed directly via pip or your favorite package manager.
 
 For educational users, an IDE from `Jet Brains <https://www.jetbrains.com/student/>`_ is available for free.
```

Comment:
Recommend removing this line.  Again, why are we using the docs to push people to IDEs?